modules/govuk_jenkins/templates/jobs/trigger_data_sync_complete.yaml.erb

```diff
@@ -8,8 +8,11 @@
         numToKeep: 10
     builders:
         - shell: |
+            echo "Getting crumb..."
+            CRUMB=$(curl https://ci_alphagov:$API_KEY@$HOSTNAME/crumbIssuer/api/json | jq '. | .crumb' | sed 's/"//g')
```

Comment:
Adding `--raw-output` to `jq` would probably allow you to get rid of `sed` here.

Reply:
Thanks, really useful!